api/lang/completion_test.go

```diff
@@ -372,112 +402,14 @@ func TestIsInStepCompletionAtEndOfLine(t *testing.T) {
 	}
 }
 
-func TestIsInStepCompletionWithParams(t *testing.T) {
-	if isStepCompletion(`* Step with "static" and <dynamic> params`, 13) {
-		t.Errorf("isStepCompletion not recognizing step context")
+func TestIsInParamContext(t *testing.T) {
```

Comment:
instead of using numbers such as `13` / `24`/ `28` to determine parameter position, can we get it via index? makes the test more readable. Something like

```
stepText := `* Step with "static" and <dynamic> params`
staticParamPosition := strings.Index(`* Step with "static" and <dynamic> params`, `"`)  + 1
if !inParameterContext(`* Step with "static" and <dynamic> params`, staticParamPosition) {
    t.Errorf("inParameterContext not recognizing step context")
}
```

Could also make it table driven, if it reduces verbosity.